server.js

```diff
@@ -149,6 +149,34 @@ async function onRequest(req,res) {
 			}
 		}
 
+		// check if the request url is using "auth-worker.js"
+		// when I just looked for "auth-worker.js" it didn't seem to actually find the url
+		if (req.url == "/js/auth-worker.js") {
+			res.writeHead(200, {
+				...HSTSHeader,
+				"Content-Security-Policy":
+			[
+				`default-src ${[
+					"'self'",
+				].join(" ")};`,
+	
+				`style-src ${[
+					"'self'",
+					"'unsafe-inline'",
+				].join(" ")};`,
+	
+				`script-src ${[
+					"'self'",
+					"wasm-unsafe-eval",
+				].join(" ")};`,
+	
+				`connect-src ${[
```

Comment:
need to verify if the web worker does any remote communication where it needs this policy. if not, let's remove it.

Reply:
I've tried looking up kind of how to verify if the web worker does a remote connection, but I have been able to find an answer that makes sense to me and applicable here. How would you say is the best way to verify this?

Reply:
I guess removing the `connect-src` from the policy and trying to see if it runs without error or not? :)

Reply:
Solid point 😅 I had done that and didn't notice any change with or without it so then I was trying to look into it more. But just removed it again and ran it and I am not seeing any errors without it so I'm going to remove it.